Packs/ExabeamSecurityOperationsPlatform/Integrations/ExabeamSecOpsPlatform/ExabeamSecOpsPlatform.yml

```diff
@@ -0,0 +1,75 @@
+category: Analytics & SIEM
+commonfields:
+  id: ExabeamSecOpsPlatform
+  version: -1
+configuration:
+- defaultvalue: https://example.com/
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: Client ID
+  name: credentials
+  required: true
+  section: Connect
+  type: 9
+  displaypassword: Client Secret
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+description: Exabeam Security Operations Platform offers robust search functionality, enabling security teams to efficiently query, analyze, and visualize security data. It provides a centralized and scalable platform for log management, threat detection, and incident investigation.
+display: Exabeam Security Operations Platform
+name: ExabeamSecOpsPlatform
+script:
+  commands:
+  - arguments:
+    - defaultValue: "7 days ago"
+      description: The starting date for the search range.
+      name: start_time
+    - defaultValue: "today"
+      description: The ending date for the search range.
+      name: end_time
+    - defaultValue: ""
+      description: Query, using Lucene syntax, filters log data for precise analysis.
+      name: query
+      required: false
+    - description: Comma-separated list of fields to be returned from the search.
+      name: fields
+      required: false
+    - description: Comma-separated list of fields by which to group the results.
+      name: group_by
```

Comment:
If this is an array an the isArray key to the argument (look it up in the repo).